tests/ext/pcntl/pcntl_fork_short_running_multiple.phpt

```diff
@@ -0,0 +1,33 @@
+--TEST--
+Short running multiple forks
+--FILE--
+<?php
+
+require 'functions.inc';
+
+const NUMBER_OF_CHILDREN = 5;
+
+call_httpbin();
+
+$currentIteration = 0;
+while ($currentIteration !== NUMBER_OF_CHILDREN) {
+    $forkPid = pcntl_fork();
+    if ($forkPid > 0) {
+        // Main
+        call_httpbin();
+    } else if ($forkPid === 0) {
+        // Child
+        call_httpbin();
+        exit(0);
+    } else {
+        exit(-1);
+    }
+
+    $currentIteration++;
+}
+
+pcntl_wait($childStatus);
+
+call_httpbin();
+?>
+--EXPECTF--
```

Comment:
Do you mind if we add: `echo 'Done.' . PHP_EOL;` at the end all the tests that don't have output? That will ensure the test didn't crash/zend_bailout.

Reply:
Done https://github.com/DataDog/dd-trace-php/pull/1196/commits/e4808a5be30fb317bc7ee5a28ab1fe0308e4930f